packages-native/safe-area-view/src/SafeAreaView.tsx

```diff
@@ -10,10 +10,8 @@ export const SafeAreaView = (props: SafeAreaViewProps<SafeAreaViewStyle>): JSX.E
     const styles = flattenStyles(defaultSafeAreaViewStyle, customStyles);
 
     return (
-        <Fragment>
-            <ReactSaveAreaView style={{ flex: 0, backgroundColor: styles.unsafeAreaTop.backgroundColor }} />
-            <ReactSaveAreaView style={{ ...styles.container, flex: 1 }}>{props.content}</ReactSaveAreaView>
-            <ReactSaveAreaView style={{ flex: 0, backgroundColor: styles.unsafeAreaBottom.backgroundColor }} />
-        </Fragment>
+        <ReactSaveAreaView style={{ flex: 1 }}>
+            <View style={{ ...styles.container }}>{props.content}</View>
```

Comment:
why add a view save area is already a view? https://github.com/facebook/react-native/blob/master/Libraries/Components/SafeAreaView/SafeAreaView.js#L41

Reply:
Because of styling reasons. Discussed it with Wesley.